public/locales/en-US/send.ftl

```diff
@@ -119,7 +119,7 @@ addPassword = Protect with password
 emailPlaceholder = Enter your email
 # $size is the size of the file, displayed using the fileSize message as format (e.g. "2.5MB")
 signInSizeBump = Sign in to send up to { $size }
-signInButton = Sign in/up
```

Comment:
we need to change the id in order for it to get translated. Let's use `signInOnlyButton` and update the js files that use it. Also, there's no need to change en-GB and en-CA files.

Reply:
If I change the id of the string then, I must change the id of other languages (other .ftl files) otherwise the translation will break (for other languages) right? Or I misunderstood some code?

Reply:
Just change the ID in en-US. Other locales will catch up in their own time, and the code will fall back to English in the meantime.

Reply:
> there's no need to change en-GB and en-CA files.

Still need to revert changes to other en-* locales. **Only** change en-US